src/Makefile

```diff
@@ -362,9 +362,8 @@ ifneq (,$(filter ${DEBUG},YES yes y Y 1))
 	OSSEC_CFLAGS+=-g
```

Comment:
Why are you stripping the `-DNDEBUG` flag?
also consider this code simplification of entire `if` block:
```makefile
OSSEC_CFLAGS+=-g
AR_LDFLAGS+=-g

ifeq (,$(filter ${DEBUG},YES yes y Y 1))
	OSSEC_CFLAGS+=-DNDEBUG
	OFLAGS+=-O2
	AR_LDFLAGS+=-s
endif
```

Reply:
Done in [Restore optimizations](https://github.com/wazuh/wazuh/pull/22011/commits/a22e559773120d81709285d5710864269afa0923) commit